### ISSUE TITLE
tf_2.x 버전 7장 decay learning rate 소스코드 관련 의견입니다.

### ISSUE THREAD

**Issue by hgstyler** [deeplearningzerotoall/TensorFlow#34]
이제 막 시작하는 단계에서 강의자료 및 깃허브 자료는 정말 잘 보고 있습니다.  

7장에서 보다 나은 학습방법을 위한 몇 가지 팁을 전달하고 계신데요,
[Learning rate decay 기법을 이용한 소스코드의 tf 2.x 버전](https://github.com/deeplearningzerotoall/TensorFlow/blob/feab13059b6c5a674c50c7c2d22dc62c7e157934/tf_2.x/lab-07-3-linear_regression_eager%20(preprocess,%20decay,%20l2_loss).ipynb) 의 마지막 출력줄에 대한 의견입니다.
구버전에서는 마지막 출력줄에 decayed learning rate를 출력하셨는데 2.x 버전에서는 빠져 있더라고요.  
  

기존 코드의 마지막줄인 아래내용을,
>  print("Iter: {}, Loss: {:.4f}".format(step, loss_value))

아래와 같이 수정하면 decayed learning rate를 출력하게 해봤습니다.
> print("Iter: {}, Loss: {:.4f}, Learning Rate: {:.8f}".format(step, loss_value, learning_rate(step)))
  

이번 장의 최종 결과물 같은 것인데 빠져있어서 의견 남겨보았습니다.  

제가 수정한 부분이 맞는 것인지 확인부탁드리고, 반영도 되었으면 좋겠습니다!
